packages/protocol/test/governance/epochrewards.ts

```diff
@@ -0,0 +1,569 @@
+      })
+    })
+  })
+
+  describe.only('#getRewardsMultiplier()', () => {
+    const timeDelta = YEAR.times(10)
+    const expectedTargetTotalSupply = getExpectedTargetTotalSupply(timeDelta)
+    const expectedTargetRemainingSupply = SUPPLY_CAP.minus(expectedTargetTotalSupply)
+    const targetEpochReward = new BigNumber(120397694238746)
+    beforeEach(async () => {
+      await timeTravel(timeDelta.toNumber(), web3)
+    })
+
+    describe('when the target supply is equal to the actual supply after rewards', () => {
+      beforeEach(async () => {
+        await mockGoldToken.setTotalSupply(expectedTargetTotalSupply.minus(targetEpochReward))
+      })
+
+      it('should return one', async () => {
+        assertEqualBN(await epochRewards.getRewardsMultiplier(targetEpochReward), toFixed(1))
+      })
+    })
+
+    describe('when the actual remaining supply is 10% more than the target remaining supply after rewards', () => {
+      beforeEach(async () => {
+        const actualRemainingSupply = expectedTargetRemainingSupply.times(1.1)
+        const totalSupply = SUPPLY_CAP.minus(actualRemainingSupply)
+          .minus(targetEpochReward)
+          .integerValue(BigNumber.ROUND_FLOOR)
+        await mockGoldToken.setTotalSupply(totalSupply)
+      })
+
+      it('should return one plus 10% times the underspend adjustment', async () => {
+        const actual = fromFixed(await epochRewards.getRewardsMultiplier(targetEpochReward))
+        const expected = new BigNumber(1).plus(
+          fromFixed(rewardsMultiplierAdjustments.underspend).times(0.1)
+        )
+        // Assert equal to 9 decimal places due to fixidity imprecision.
+        assert.equal(expected.dp(9).toFixed(), actual.dp(9).toFixed())
+      })
+    })
+
+    describe('when the actual remaining supply is 10% less than the target remaining supply after rewards', () => {
+      beforeEach(async () => {
+        const actualRemainingSupply = expectedTargetRemainingSupply.times(0.9)
+        const totalSupply = SUPPLY_CAP.minus(actualRemainingSupply)
+          .minus(targetEpochReward)
+          .integerValue(BigNumber.ROUND_FLOOR)
+        await mockGoldToken.setTotalSupply(totalSupply)
+      })
+
+      it('should return one minus 10% times the underspend adjustment', async () => {
+        const actual = fromFixed(await epochRewards.getRewardsMultiplier(targetEpochReward))
+        const expected = new BigNumber(1).minus(
+          fromFixed(rewardsMultiplierAdjustments.overspend).times(0.1)
+        )
+        // Assert equal to 9 decimal places due to fixidity imprecision.
+        assert.equal(expected.dp(9).toFixed(), actual.dp(9).toFixed())
+      })
+    })
+  })
+
+  describe.only('#updateTargetVotingYield()', () => {
+    const randomAddress = web3.utils.randomHex(20)
+    const totalSupply = new BigNumber(129762987346298761037469283746)
+    const reserveBalance = new BigNumber(2397846127684712867321)
```

Comment:
Could you note whether or not these are arbitrary? (So future editors know how important it is to preserve them)